released/packages/coq-haskell/coq-haskell.1.1.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "johnw@newartisans.com"
+version: "1.1.0"
+
+homepage: "https://github.com/jwiegley/coq-haskell"
+dev-repo: "git+https://github.com/jwiegley/coq-haskell.git"
+bug-reports: "https://github.com/jwiegley/coq-haskell/issues"
+license: "BSD-3-Clause"
+
+synopsis: "A library to provide Haskell-familiar constructions in Coq"
+description: """
+This library is designed for Haskell users who are either using Coq to build
+code intended for extraction to Haskell, or who wish to prototype/prove their
+algorithms in Coq. It provides a collection of definitions and notations to
+make Gallina more familiar to Haskellers.
+"""
+
+build: [make "JOBS=%{jobs}%" ]
+install: [make "install"]
+depends: [
+  "coq" {(>= "8.14" & < "8.16~") | (= "dev")}
```

Comment:
```suggestion
  "coq" {(>= "8.14" & < "8.17~") | (= "dev")}
```
The package works well on Coq 8.16+rc1, so I think we can say it's safe for the yet-to-be-released Coq 8.16.0